src/bindings/capi/thorvg_capi.h

```diff
@@ -27,12 +27,15 @@
 #endif
 
 #ifdef TVG_BUILD
-    #define TVG_EXPORT __attribute__ ((visibility ("default")))
+    #ifdef _WIN32
```

Comment:
In `inc/thorvg.h` we are using `_MSC_VER` instead of `_WIN32`.
How about using `_MSC_VER`?

Reply:
That would be fine :) I copied the `_WIN32` example from an existing file `tvgXmlParser` but I am happy to use whatever is suggested.

Reply:
@hermet Do you think `#ifdef _WIN32` should be changed to `#ifdef _MSC_VER`?
On windows:
- gcc supports both  `__attribute__ ((visibility ("default")))` and `__declspec(dllexport)`
- clang supports both `__attribute__ ((visibility ("default")))` and `__declspec(dllexport)`
- MSVC only supports `__declspec(dllexport)`
So they both work fine. What is your preference?